pkgs/applications/misc/deco/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchFromGitHub, scsh, feh, xlibs }:
```

Comment:
I did that, but I got: `error: anonymous function at /pub/nixpkgs/pkgs/applications/misc/deco/default.nix:1:1 called with unexpected argument ‘xdpyinfo’, at /pub/nixpkgs/lib/customisation.nix:56:12`